torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -50,8 +51,27 @@ TensorView::TensorView(const std::shared_ptr<c10::TensorType>& tensor_type)
       sizes.push_back(new IterDomain(new Int(0), new Int()));
     }
   }
-  domain_ = new TensorDomain(sizes);
 
+  for (decltype(tensor_type->dim().value()) i = 0;
+       i < tensor_type->dim().value();
+       i++) {
+    // TODO: this is a temporary WAR to avoid contiguous_ flag on broadcasted
```

Comment:
This should be fixed now in https://github.com/csarofeen/pytorch/pull/225 but will include this and other fixes there after we merge this.